### DISCUSSION TITLE
BUG in STATIC DEPLOY

### DISCUSSION THREAD

**Discussion by webkooid** [vercel/community#4064] [Help]
### Page to Investigate

https://webkoo.id/nextjs/

### Steps to Reproduce

Hello, this BUG happen today

You just need to Install Next JS
Then edit next.config.js
Just like at https://nextjs.org/docs/pages/building-your-application/deploying/static-exports

Then, run in "OUT" folder,
ERROR then, JS, CSS, Image, not found

Please Fix this problem, its very annoying

**Comment by amyegan** [comment]
Hey @webkooid. I wasn't able to replicate this error. I tried creating a new project using `create-next-app`, changed the config, build the static export, and deployed it. I didn't see any errors.

Do you have a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) so we can work together to debug this?
